### PR TITLE
[docs]Add Unified Cloudwatch Agent Images to README

### PR DESCRIPTION
**Pull request by Zyiqin-Miranda**
## Why are these changes needed?

This PR adds Images with Unified Cloudwatch Agent to amazon-ray README.md.

## Checks

- [ ] I'm working against the latest source on the **experimental** branch.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Latest Amazon Ray wheels:
 
 All of the above wheels have passed unit tests. They can be installed via `pip install -U [wheel URL]`.
 
-### Images
+### Amazon-Ray Images
```
Comment (pdames) [inline README.md:27]:
```suggestion
### Amazon Ray Images
```

File: README.md [hunk 2]
```diff
@@ -129,6 +129,17 @@ setup_commands:
   - sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -m ec2 -a stop
 ```
 
+### Unified CloudWatch Agent Images
+The AMI below is based on version 48 of the Ubuntu 18.04 AWS Deep Learning AMI and ships with Unified CloudWatch Agent v1.247348.0b251302.  
+Note that `T6Iq2faj` suffix in AMI Name serves as a unique identifier indicating Unified CloudWatch Agent is installed.  
+EC2 AMIs with Unified CloudWatch Agent pre-installed:
```
Comment (pdames) [inline README.md:135]:
```suggestion
The AMIs below ship with the [Unified CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/UseCloudWatchUnifiedAgent.html) pre-installed. They should be used with any Ray cluster declaring an `agent` configuration under the `cloudwatch` section of its AWS [cluster config file](https://docs.ray.io/en/latest/cluster/config.html).
```

File: README.md [hunk 3]
```diff
@@ -129,6 +129,17 @@ setup_commands:
...
+
+| Base AMI                                     | AMI ID                | Region    |
```
Comment (pdames) [inline README.md:137]: Should we add a new column showing the Unified CloudWatch Agent version installed on each AMI?

**Review by pdames — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (annaluo676) [reply README.md:137]: Nit: shall we include corresponding ray version as well, or do the published AMIs always come with the ray wheels listed above?

Comment (Zyiqin-Miranda) [reply README.md:137]: > Nit: shall we include corresponding ray version as well, or do the published AMIs always come with the ray wheels listed above?

Thanks Anna, ray is not installed on our CloudWatch agent AMIs. That way we do not have to worry about keep in sync with latest ray versions.

**Review by pdames — APPROVED** [review]

**Pull request merged**